src/services/NodeReducerService.ts

```
@@ -281,100 +255,147 @@ export class NodeDupReduction {
+            if (!similarNodes || similarNodes.length === 0) {
+              return { merged: 0, sourceId };
+            }
+
+            for (const similarNode of similarNodes) {
+              const { nodeId: targetId, nodeName: targetName } = similarNode;
+
+              alts.push({ id: targetId, name: targetName });
+
+              if (!this.verbose) {
+                await this.migrateNodeRelations(sourceId, targetId);
+                if (nodesObj.hasOwnProperty(targetId)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)